test/cppsuite/test_harness/workload/random_generator.cxx

```diff
@@ -37,25 +42,24 @@ random_generator::instance()
 }
 
 std::string
-random_generator::generate_string(std::size_t length)
+random_generator::generate_string(std::size_t length, characters_type type)
 {
-    std::string random_string;
-
-    for (std::size_t i = 0; i < length; ++i)
-        random_string += _characters[_distribution(_generator)];
-
-    return (random_string);
+    std::string str(get_characters(type));
+    std::shuffle(str.begin(), str.end(), _generator);
+    return (str.substr(0, length));
 }
 
 std::string
-random_generator::generate_pseudo_random_string(std::size_t length)
+random_generator::generate_pseudo_random_string(std::size_t length, characters_type type)
```

Comment:
As far as I understand this method can only generate as many unique strings of a given length as the number of elements in `characters`.  And it's 26 or 62 depending on `characters_type`. Is that the expectation from this method?

Reply:
Yes that's correct.

Reply:
I added a comment that it should not be used for key generation as part of  b1683ad4ba6f28c85b27429ca517e649b3b440cf